benchcab/config.py

```diff
@@ -67,6 +67,28 @@ def validate_config(config: dict) -> bool:
     raise ConfigValidationException(v)
 
 
+def read_optional_data(config: dict):
+    if "realisations" in config:
+        for r in config["realisations"]:
+            r["name"] = r.get("name")
+
```

Comment:
This does not do anything so remove.

Reply:
We use it in making `Path` before initialising `Repo` here:

https://github.com/CABLE-LSM/benchcab/blob/0af9e4e6751a4c0adbaeccba88994697b6b9f8f4/benchcab/benchcab.py#L111

Reply:
Oh right, sorry, my mistake when reading the code. Ignore.